examples/mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+plugins = axion.mypy, pydantic.mypy
```

Comment:
Yeah, it uses the plugin which is being developed on top of `example` to see how much of `handler` runtime analysis can be replaced with static type checking.